src/libNetwork/P2PComm.cpp

```diff
@@ -771,6 +977,236 @@ void P2PComm::AcceptConnectionCallback([[gnu::unused]] evconnlistener* listener,
   bufferevent_enable(bev, EV_READ | EV_WRITE);
 }
 
+void P2PComm::RemoveBevFromMap(const Peer& peer) {
+  LOG_MARKER();
```

Comment:
Maybe you could call this only if log level is DEBUG

Reply:
Removed `LOG_MARKER` as we are already printing debug inside function.